addons/mail/static/src/core/public_web/thread_model_patch.js

```diff
@@ -36,9 +36,6 @@ patch(Thread.prototype, {
      * @param {import("models").Message} message
      */
     notifyMessageToUser(message) {
-        if (this.isCorrespondentOdooBot) {
-            return;
-        }
```

Comment:
Note: this was just way too drastic and prevented any notif from OdooBot, even in other potentially important flows.
It completely shadowed condition in `discuss.channel/new_message` too.

We might need to backport this part actually.